config/sync/govuk_design_system.settings.yml

```diff
@@ -0,0 +1,15 @@
+phase_banner: ''
+phase_banner_message:
+  value: "<p>This is a new service – your <a class=\"govuk-link\" href=\"/contact/feedback\">feedback</a> will help us to improve it.</p>\r\n"
```

Comment:
That feedback URL was dead for me. Should we link to the public Slack channel (or use a `mailto:`) as an interim?

Reply:
We're not actually using the phase banner, the `phase_banner: ''` line means it's turned off. (We're not in an Alpha/Beta phase so doesn't make sense to have one).
That text value is just what comes with the theme, it's the default value.

Reply:
:+1:, `''` is wonderfully intuitive for turning something off 🤪. Thanks, this is fine in that case 🙌